src/SFA.DAS.FindEpao/InnerApi/Responses/GetEpaoCourseListItem.cs

```diff
@@ -8,5 +8,6 @@ public class GetEpaoCourseListItem
         public DateTime? EffectiveFrom { get; set; }
         public DateTime? EffectiveTo { get; set; }
         public DateTime? DateStandardApprovedOnRegister { get; set; }
+        public GetStandardsExtendedListResponse StandardVersions { get; set; }
```

Comment:
Can this not be changed to a `IEnumerable<GetStandardsExtendedListeItem>> ? not sure we need the extra object.

Reply:
removed extra object & changed to IEnumerable